src/Apps/Collect/CollectApp.tsx

```diff
@@ -18,11 +18,23 @@ export interface CollectAppProps {
 
 export class CollectApp extends Component<CollectAppProps> {
   render() {
+    const collection = {
+      id: "minimalist-prints",
+      title: "minimalist prints",
+      image:
+        "https://d7hftxdivxxvm.cloudfront.net?resize_to=width&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2Fbzx0Po9dHb_GhcvbD_c7jQ%2Funtouched-jpg.jpg&width=2000&quality=80",
+      description:
+        "Brian Donnelly, better known as KAWS, spent the first year of his career as an animator for Disney. After leaving in 1997, KAWS took inspiration from the company’s signature cartoon, Mickey Mouse, to create his own set of characters that he named “Companions.” With gloved hands and X’s for eyes, “Companions” first appeared in KAWS’s graffiti works across New York City in the late 1990s. By the end of the decade, the street artist created his first three-dimensional version, and his characters have since taken on a variety of colors, sizes, and poses. In 2017, his four-foot-tall Seated Companion (2011) broke the auction record for the series, selling for over $400,000. However, many of KAWS’s “Companions” are considerably more affordable, such as his vinyl toys produced in collaboration with esteemed manufacturers including Bounty Hunter, Bape, Medicom, and his own brand OriginalFake, which was active between 2006 and 2013.",
+      medium: "prints",
+      major_periods: [],
+      gene_ids: [],
+      artist_ids: [],
+    }
     return (
       <Fragment>
         <Flex flexDirection="column">
           {true ? (
-            <CollectionHeader />
+            <CollectionHeader collection={collection} />
```

Comment:
It seems the work flow you were using was just to render the Collections Header inside of the CollectApp. I assume once everything is hooked up we will just move things into `/Collect/routes.tsx`?

Reply:
Right, that's the route I was thinking initially but on second though, I think we should create a new top level app component for Collections (`CollectionApp`) for simplicity sake since the root query will be different. So we'd end up with

```
Apps/Collect
|- CollectApp.tsx
|- CollectionApp.tsx
|- routes.tsx
```

Reply:
Okay I will refactor to go with the new approach.